<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LC_ModifyGapOptions</class>
 <widget class="QWidget" name="LC_ModifyGapOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>24</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item row="0" column="1">
    <widget class="QLineEdit" name="leSize">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Gap Snap:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Snap Distance:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QCheckBox" name="cbFree">
     <property name="text">
      <string>Free</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QLineEdit" name="leDistance">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Gap Size:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QComboBox" name="cbLineSnap">
     <item>
      <property name="text">
       <string>Free</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Start</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Middle</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>End</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="9">
    <widget class="QComboBox" name="cbGapSnap">
     <item>
      <property name="text">
       <string>Start</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Middle</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>End</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Line Snap:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="10">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
